<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JAProjektClass</class>
 <widget class="QWidget" name="JAProjektClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>JAProjekt</string>
  </property>
  <widget class="QPushButton" name="loadFileButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Load file</string>
   </property>
  </widget>
  <widget class="QLabel" name="sourceLocationLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>431</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cppAlgButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>C++ algorithm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="asmAlgButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>40</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Assembly algorithm</string>
   </property>
  </widget>
  <widget class="QLabel" name="cppAlglabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="asmAlglabel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>110</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveFileButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>360</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save file</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="JAProjekt.qrc"/>
 </resources>
 <connections/>
</ui>
